app/views/users/settings.html.erb

```diff
@@ -90,8 +94,45 @@
   <br />
   <br />
 
+  <% if current_user.can_moderate? %>
+    <div style="display: inline-flex; justify-content: space-between; width: 90%;">
+      <span>In a weekly digest for unmoderated posts</span>
+      <span>
+        <label style=" vertical-align: middle;" class="switch">
+          <p> Notification switch </p>
+          <input type="checkbox" name="digest:weekly:spam" <% if UserTag.exists?(current_user.id, 'digest:weekly:spam') %>checked<% end %>>
+          <span class="slider round"></span>
+        </label>
+      </span>
+    </div>
+
+    <br />
+    <br />
+
+    <div style="display: inline-flex; justify-content: space-between; width: 90%;">
+      <span>In a daily digest for unmoderated posts</span>
+      <span>
+        <label style=" vertical-align: middle;" class="switch">
+          <p> Notification switch </p>
+          <input type="checkbox" name="digest:daily:spam" <% if UserTag.exists?(current_user.id, 'digest:daily:spam') %>checked<% end %>>
+          <span class="slider round"></span>
+        </label>
+      </span>
+    </div>
+
+    <br />
+    <br />
+  <% end %>
+
+  <br />
+
```

Comment:
Some of these lines differ from what was there previously, is this also part of the issue changes?

Reply:
@RuthNjeri the changes mentioned in the issue are just for when we login as a "user". I've extended it for "moderator" and "admin" logins too to maintain uniformity. Hence those changes.
Thanks!